sdk/tables/azure-tables/README.md

```diff
@@ -0,0 +1,68 @@
+# Azure Tables client library for JavaScript
+
+[Azure Tables](https://azure.microsoft.com/en-us/services/storage/tables/) is a cloud-based service that stores structured NoSQL data, providing a key/attribute store with a schemaless design. Tables storage gives developers flexibility and scalability with all the best parts of Azure cloud.
```

Comment:
It looks like the docs refer to data as `semi-structured`, should this be instead `[...] is a cloud-based service that stores semi-structured NoSQL data`?

Reply:
I got this from the [official docs](https://docs.microsoft.com/en-us/azure/cosmos-db/table-storage-overview) where it refers to them as structured data